composer.json

```diff
@@ -32,13 +32,13 @@
     },
     "require-dev": {
         "doctrine/doctrine-bundle": "^2.0",
-        "doctrine/mongodb-odm": "~1.0",
+        "doctrine/mongodb-odm": "^2.2.0",
         "doctrine/orm": "~2.2",
         "phing/phing": "~2.4",
         "php-mock/php-mock-phpunit": "~1.0 || ~2.0",
+        "phpstan/phpstan": "^0.12",
         "phpstan/phpstan-phpunit": "~0.9",
-        "phpstan/phpstan-shim": "~0.9",
-        "phpunit/phpunit": "~5.0 || ~6.0",
+        "phpunit/phpunit": "~9.0",
```

Comment:
This won't support PHP 7.2 as `phpunit/phpunit:^9.0` requires PHP 7.3 , but as CI is broken, I'm fine with it.
Need to spend some time to migrate CI to github action and make it work again.

Reply:
I had to in order to use php 8 sadly :/ Github action work nicely and I got some experience. Could help, feel free to reach me :)